repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -107,12 +107,19 @@ class Interpreter(prompt0: Ref[String],
 
   def processModule(code: String, fileName: String = "Main.scala") = processScript(
     skipSheBangLine(code),
-    (code, imports) => withContextClassloader(eval.processScriptBlock(code, imports, printer, fileName))
-    )
+    (code, imports) => withContextClassloader(eval.processScriptBlock(code,
```

Comment:
Not sure if this is a good formatting style but I found it most natural to avoid too long lines.